content/docs/self-hosted-runners.md

```diff
@@ -148,19 +148,23 @@ For example, `docker://iterativeai/cml:0-dvc2-base1-gpu`, or
 The `cml runner` command supports many options (see the
 [command reference](/doc/ref/runner)). Notable options are:
 
-- `--labels`: One or more comma-delimited labels (e.g. "cml,gpu").
-- `--idle-timeout`: Seconds to wait for jobs before shutting down.
-- `--single`: Terminate after running a single job.
+- `--labels=<...>`: One or more (comma-delimited) labels (e.g. `cml,gpu`).
+- `--idle-timeout=<seconds>`: Seconds to wait for jobs before shutting down.
+- `--single`: Shutdown runner after one workflow run.
```

Comment:
```suggestion
- `--single`: Shutdown runner after one workflow run.
```

Even “quit” or “exit” perhaps?

Reply:
Neither "quit" nor "exit" sound quite right (PS: I see an empty diff in your suggested change). How about

```suggestion
- `--single`: Terminate runner after one workflow run.
```

Reply:
> _I see an empty diff in your suggested change_

Agh! Edited to remove the “suggestion” block

Reply:
“Terminate” sounds a bit like AWS EC2 instances, but it's not necessarily tied to those, much less for on-premises scenarios. What about “remove” or “unregister”?

Reply:
When providing this option, runners will take a single job and exit, where exiting implies unregistering themselves.

Reply:
🚲 🅿️ 🤷🏼‍♂️

Reply:
seems important though... Scripts/jobs can exit/quit, but instances/machines can terminate/shut down. Seems more important to mention the termination.

Reply:
Termination is exclusive to `cml runner --cloud`